src/PhpSpec/Console/ContainerAssembler.php

```diff
@@ -18,29 +18,30 @@
 use PhpSpec\CodeAnalysis\TokenizedNamespaceResolver;
 use PhpSpec\CodeAnalysis\TokenizedTypeHintRewriter;
 use PhpSpec\CodeAnalysis\VisibilityAccessInspector;
+use PhpSpec\CodeGenerator;
+use PhpSpec\Config\OptionsConfig;
 use PhpSpec\Console\Assembler\PresenterAssembler;
 use PhpSpec\Console\Prompter\Question;
 use PhpSpec\Factory\ReflectionFactory;
-use PhpSpec\Process\Prerequisites\SuitePrerequisites;
-use PhpSpec\Util\ClassFileAnalyser;
-use PhpSpec\Util\Filesystem;
-use PhpSpec\Util\ReservedWordsMethodNameChecker;
-use PhpSpec\Process\ReRunner;
-use PhpSpec\Util\MethodAnalyser;
-use Symfony\Component\EventDispatcher\EventDispatcher;
-use PhpSpec\ServiceContainer\IndexedServiceContainer;
-use PhpSpec\CodeGenerator;
 use PhpSpec\Formatter as SpecFormatter;
 use PhpSpec\Listener;
 use PhpSpec\Loader;
 use PhpSpec\Locator;
 use PhpSpec\Matcher;
+use PhpSpec\Message\CurrentExampleTracker;
+use PhpSpec\NamespaceProvider\Factory;
+use PhpSpec\Process\Prerequisites\SuitePrerequisites;
+use PhpSpec\Process\ReRunner;
+use PhpSpec\Process\Shutdown\Shutdown;
 use PhpSpec\Runner;
+use PhpSpec\ServiceContainer\IndexedServiceContainer;
+use PhpSpec\Util\ClassFileAnalyser;
+use PhpSpec\Util\Filesystem;
+use PhpSpec\Util\MethodAnalyser;
+use PhpSpec\Util\ReservedWordsMethodNameChecker;
 use PhpSpec\Wrapper;
-use PhpSpec\Config\OptionsConfig;
+use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Component\Process\PhpExecutableFinder;
-use PhpSpec\Message\CurrentExampleTracker;
-use PhpSpec\Process\Shutdown\Shutdown;
```

Comment:
My editor reorder the statements, I put that change in a separate commit in case you guys don't want it.